src/utils/io.ts

```diff
@@ -239,6 +239,20 @@ export async function writeDeployedContractToFile(
   });
 }
 
+// Read existing accounts into env
+export async function readAccountsIntoEnv(
+  env: any,
+  accountRegistry: AccountRegistry
+) {
+  console.log(env, accountRegistry.keys());
```

Comment:
pub keys right?

Reply:
err whoops, yes it only logs pub keys, but I should get rid of this - good catch!

Reply:
removed now!